CPP_STYLE_GUIDE.md

```diff
@@ -21,6 +21,7 @@
   * [Use explicit pointer comparisons](#use-explicit-pointer-comparisons)
   * [Ownership and Smart Pointers](#ownership-and-smart-pointers)
   * [Avoid non-const references](#avoid-non-const-references)
+  * [When to use AliasedBuffers](#when-to-use-aliasedbuffers)
```

Comment:
Maybe `Use AliasedBuffers to manipulate TypedArrays` would be easier to recognize in the TOC?

Reply:
@joyeecheung - done, thanks.